_sass/core/cryostat.scss

```diff
@@ -0,0 +1,133 @@
+
+.logo-title {
+  width: 100%;
+  max-width: 200px;
+}
+
+.cryostat-header {
+  margin-top: .5em;
+  margin-bottom: .5em;
+}
+
+.site-nav {
+  margin-top: .25em;
+}
+
+.footer-heading {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 150px;
+}
+
+.cryostat-heading-1 {
+  @extend .page-heading;
+}
+
+.cryostat-blurb {
+  display: inline-block;
+  text-align: center;
+  margin-top: 2em;
+}
+
+.cryostat-blurb-left {
+  float: left;
+  width: 40%;
+  font-size: 2.5em;
```

Comment:
If the size is 2.3em then the title will fit on 3 lines instead of 4

Reply:
I also wonder how it would look if both blurbs were centered on top of each other. I was looking at Patternfly's landing page for inspiration and they stack the text such that your eyes would read the blurbs from top to bottom instead of left to right:
![image](https://user-images.githubusercontent.com/84587295/134712801-75878872-a2e7-43bb-be80-0b381871334c.png)

Reply:
I'm a fan of that design as well. 

Though the inspiration here I believe is the quarkus.io page that also uses Jekyll for their website, and they chose the left-right hero text orientation as well.

I think a top-bottom approach might be better for smaller screen sizes as well, as we get a bit of overlap. The quarkus page gets spaced out pretty funny on my phone as well.
![2021-09-24-122205_352x597_scrot](https://user-images.githubusercontent.com/10425301/134713504-73e7e3b8-34f8-4e16-b292-c7f1bfc1ec33.png)

Reply:
This can definitely be tweaked. I was hoping to do left-right and then when it's smaller go top-bottom, but I need to learn more CSS first :|

Reply:
https://css-tricks.com/wp-content/uploads/2021/04/css-is-awesome.jpg

Reply:
That's some advanced css.

Just an additional thought on the front-page, in the absence of a proper image you could still add a plain coloured background to create the illusion of a banner. 

Something roughly like:
![2021-09-24-173509_1916x1036_scrot](https://user-images.githubusercontent.com/10425301/134742320-2a2b5ec3-914b-4618-b174-ca66cf6ad6e0.png)